x-pack/platform/plugins/shared/actions/server/routes/connector/get_all/get_all.ts

```diff
@@ -26,10 +29,19 @@ export const getAllConnectorsRoute = (
         access: 'public',
         summary: `Get all connectors`,
         tags: ['oas-tag:connectors'],
-        // description:
-        //   'You must have `read` privileges for the **Actions and Connectors** feature in the **Management** section of the Kibana feature privileges.',
       },
-      validate: {},
+      validate: {
+        request: {},
+        response: {
+          200: {
+            body: () => connectorWithExtraFindDataSchemaV2,
```

Comment:
Is this type right?

It returns a single object still. This points to:

```
export const allConnectorsResponseSchema = connectorResponseSchema.extends({
  referenced_by_count: schema.number({
    ...
  }),
});

export const connectorResponseSchema = schema.object({
  ...
});
```

But the response is an array `AllConnectorsResponseV1[]`.

Reply:
Great point. I am fixing them.